.github/workflows/nv-a6000-fastgen.yml

```diff
@@ -35,8 +35,9 @@ jobs:
           python -c "import torch; print('CUDA available:', torch.cuda.is_available())"
       - name: Install transformers
         run: |
-          git clone --depth=1 https://github.com/huggingface/transformers
```

Comment:
Revert these changes, as they are only required for now in order while we wait for this PR to be merged which fixes transformers for torch<2.4

Reply:
PR to HF transformers completed.